client/components/DisplayTest/index.jsx

```diff
@@ -81,7 +82,7 @@ class DisplayTest extends Component {
                 break;
             }
             case 'goToNextTest': {
-                displayNextTest();
+                this.iframe.current.triggerSkip();
```

Comment:
I actually think this is probably not a good thing to add -- because we don't need any information from the iframe at all, we really just want to grab the serialized form and send save a "incomplete" result in the `test_result` table. I can see how you are trying to hook into the same code for saving that is used later on, but I think it would be simplier to just have a different code path that calls `SaveResult({.... status: "incomplete"})`